include/freerdp/client/rdpsnd.h

```diff
@@ -51,6 +52,8 @@ struct rdpsnd_device_plugin
 	pcStart Start;
 	pcClose Close;
 	pcFree Free;
+
+	rdpContext* rdpcontext;
```

Comment:
Not required, already part of `rdpsnd_plugin`. 
better export that in a private header only available in the sound channel, then you don´t need to change the public API.

Reply:
not sure i understood how to solve this. rdpsnd_plugin is not exported, that's why i added it

Reply:
@akallabeth ?

Reply:
sorry, forgot about this one. instead of adding an additional pointer to a struct you could just use a getter (`FREERDP_API rdpContext* freerdp_rdpsnd_get_context(rdpsndPlugin* rdpsnd);`) so you don´t change the existing structs but allow access to internal members.
that way existing plugins would not need to be recompiled for this function to work.

Reply:
got you, thanks.